db/migrate/20160510122323_index_foreign_keys_in_approvals.rb

```diff
@@ -1,6 +1,6 @@
-class IndexForeignKeysInApprovals < ActiveRecord::Migration
+class IndexForeignKeysInApprovals < ActiveRecord::Migration[5.2]
   def change
-    add_index :approvals, :edition_id
-    add_index :approvals, :user_id
+    add_index :approvals, :edition_id unless index_exists?(:approvals, :edition_id)
+    add_index :approvals, :user_id unless index_exists?(:editions, :user_id)
```

Comment:
Should this be `:approvals` not `:edition`?

When trying to apply it I get an error because `index_approvals_on_user_id` already exists.

Reply:
Good spot - it should be! 🤦

Reply:
```suggestion
    add_index :approvals, :user_id unless index_exists?(:approvals, :user_id)
```